### ISSUE TITLE
Expanding the rename input-field

### ISSUE THREAD

**Issue by erik-alm** [michitux/dokuwiki-plugin-move#210]
When using the Bootstrap3/Darkly theme I had to add the following CSS (to `conf/userstyle.css`) in order for the rename form input to expand to the full size of the move form:

```
div.plugin_move_dialog form label {
  display: block;
}
```

Not sure if this is a theme problem or a bug but the label element's `display: inline-block` seems to be the culprit.

Tested on several browsers (only macos, though) and in Greebo and Hogfather.

**Comment by phy25** [comment]
Does this happen on other templates (e.g. the default dokuwiki template)? If not, I'd say this should be an issue to the bootstrap3 template.

**Comment by erik-alm** [comment]
@phy25, no, not as far as I can tell. However, I'm guessing bootstrap is some kind of standard?
